docs/manual/docs/cmd/graph/teams/teams-user-list.md

```diff
@@ -0,0 +1,47 @@
+# graph teams user list
+
+Lists Microsoft Teams teams users for a specified team.
+
+## Usage
+
+```sh
+graph teams user list [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-i, --teamId <teamId>`|The GroupId of the team
+`-r, --role <type>`|Filter the results to only users with the given role: Owner|Member|Guest
```

Comment:
Allowed values should be wrapped in backticks. Otherwise, they break table formatting introducing new columns